cmd/tetra/main.go

```diff
@@ -42,10 +41,9 @@ func New() *cobra.Command {
 
 	addCommands(rootCmd)
 	flags := rootCmd.PersistentFlags()
-	flags.BoolP(common.KeyDebug, "d", false, "Enable debug messages")
-	flags.String(common.KeyServerAddress, "localhost:54321", "gRPC server address")
-	flags.Duration(common.KeyTimeout, 10*time.Second, "Connection timeout")
-	flags.Int(common.KeyRetries, 0, "Connection retries with exponential backoff")
-	viper.BindPFlags(flags)
+	flags.BoolVarP(&common.Debug, common.KeyDebug, "d", false, "Enable debug messages")
+	flags.StringVar(&common.ServerAddress, common.KeyServerAddress, "", "gRPC server address")
```

Comment:
ah sorry we will need a last fix, I just noticed by reading carefully that the default value `localhost:54321` value was removed here, not sure this is intentional?